src/ngraph_encapsulate_op.cc

```diff
@@ -284,6 +314,17 @@ class NGraphEncapsulateOp : public OpKernel {
       }
 
       m_ng_functions[signature] = ng_function;
+      // Memory after
+      mem_usage(vm, rss);
+      auto delta_vm_mem = vm - vm0;
+      auto delta_res_mem = rss - rss0;
+      NGRAPH_VLOG(1) << "NGTF_CACHE_PROFILE: " << ctx->op_kernel().name()
+                     << "  Step_ID: " << ctx->step_id();
+      NGRAPH_VLOG(1) << "       Delta Virtual Memory Measurment: "
+                     << delta_vm_mem
+                     << "  Delta Resident Memory Measurment: " << delta_res_mem
+                     << "  Function Memory Measurment: " << function_size;
+      NGRAPH_VLOG(1) << "       Resident Memory: " << rss;
```

Comment:
@avijit-nervana 
Should we use an environment variable for this like NGRAPH_TF_CACHE_LOGS=1 or NGRAPH_VLOG is ok?